### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Laurentjg**
- Added explanation and links to project ReLevENT
- Added links to use-cases at ReLevENT. (Note: so use-cases are not in der-scheduling-repo).

### CODE REVIEW

**Comment by Laurentjg** [comment]
@fhg-dw or @mzillgith, please check on change and handle request :-).

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 
 ## Introduction
-This project aims to create a production grade open-source scheduling software stack, enabling consumers, manufacturers and automations suppliers to implement IEC Scheduling according to IEC 61850-7-4-2010 Annex K. Formaly known as TR IEC 61850-90-10. The main use-case is for controling Distributed Energy Resources (DER), but not limited to that. 
+This project aims to create a production grade open-source scheduling software stack, enabling consumers, manufacturers and automations suppliers to implement IEC Scheduling according to IEC 61850-7-4-2010 Annex K. Formaly known as TR IEC 61850-90-10. The main use-case is for controling Distributed Energy Resources (DER), but not limited to that. The software is (being) integrated to become useful for interacting with communication protocols in [project ReLevENT](https://github.com/alliander-opensource/ReLevENT). See also here for [use-cases](https://github.com/alliander-opensource/ReLevENT/blob/main/usecases/use-cases-edge-view.md) where it can be applied to.
```
Comment (Sander3003) [inline README.md:5]: Why is the benefit of starting a new repository? RelevENT? I don't get the differences.

Comment (mzillgith) [reply README.md:5]: The idea is that the ReLevENT repository contains only the additional code for the fledge plugin that is based on the DER-Scheduler code. The DER Scheduler code that is independent from Fledge will stay in its own repository.

Comment (Laurentjg) [reply README.md:5]: @Sander3003 Yes, this is what we agreed on to have the DER-scheduling remaining the core and ReLevENT becomes an integration project.

**Pull request merged**